src/components/MediaViewer/MediaViewer.tsx

```diff
@@ -120,6 +122,19 @@ const MediaViewer = ({ resource }: { resource: CloudinaryResource }) => {
 
   const hasTransformations = !!Object.entries(transformations).length;
 
+  useKeydown({
+    key: "Escape",
+    handler: (_event) => {
+      if (hasTransformations) {
```

Comment:
Based on what I saw in the logic for the page, and pending edits will be detected via `hasTransformations`?

Reply:
yup!!